lib/internal/console/constructor.js

```diff
@@ -57,7 +57,7 @@ const { Buffer: { isBuffer } } = require('buffer');
 const {
   inspect,
   formatWithOptions,
-} = require('internal/util/inspect');
+} = require('internal/console/inspect');
```

Comment:
I'm sorry but I didn't understand the intention behind this change. Can you elaborate?

Reply:
So since `console` relies `util.inspect` and `util.format` which are not stable we should basically create a copy paste stable implementation